### ISSUE TITLE
Add "(Upcoming) Events" section to documentation front page.

### ISSUE THREAD

**Issue by josiah-wolf-oberholtzer** [Abjad/abjad#523]

**Comment by jdavancens** [comment]
@josiah-wolf-oberholtzer OK now what happens? I've just forked and cloned the master branch.

**Comment by jdavancens** [comment]
Maybe I should have just cloned /josiah/issue-543-cleanup-install-guide ?

**Comment by josiah-wolf-oberholtzer** [comment]
@jdavancens The `josiah/issue-543-cleanup-install-guide` is just for tackling issue #543. If you want to take a look at that branch, you need to switch from the `master` branch to `josiah/issue-543-cleanup-install-guide`:

    $ git checkout josiah/issue-543-cleanup-install-guide

For **this** issue, I would create a new branch. First, make sure you're on `master`:

    $ git checkout master

Then create the new branch:

   $ git checkout -b joseph/issue-523-upcoming-events

That above command means "create a new branch starting on whatever commit I was at before", so branching off of `master`, because that was the previous branch you were on.

Then you can begin making changes. You'll need to edit `abjad/docs/source/index.rst`.

**Comment by jdavancens** [comment]
Got it

**Comment by josiah-wolf-oberholtzer** [comment]
Once you make changes (or even before), you should build the docs to take a look. I gotta warn you - that first build takes a *long* time. Additionally:

- Make sure you have the development dependencies installed. Install your local copy of Abjad with dev dependencies if you haven't already done so:
    
    ```
    $ cd abjad
    abjad$ sudo pip install -e .[development,ipython]
    ...
    ```

- Make sure you have Graphviz installed: http://www.graphviz.org/. That's the tool responsible for building all of the inheritance graphs in our docs. Doc building might blow up without it.

**Comment by josiah-wolf-oberholtzer** [comment]
Closed by #559.
